changelog/v1.6.0/consul-tls-discovery.yaml

```
@@ -0,0 +1,10 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo/issues/2544
+    description: >-
```

perhaps you want
```suggestion
    description: |
```
since you have formatted whitespace in the changelog

---

I'll make the change, but out of curiosity where can I learn about this changelog syntax? I've just been trying to infer what I could from examples.

---

https://yaml-multiline.info/

---

Thanks!